app/controllers/clearance/passwords_controller.rb

```diff
@@ -56,11 +65,15 @@ def password_reset_params
     end
   end
 
-  def find_user_by_id_and_confirmation_token
-    user_param = Clearance.configuration.user_id_parameter
+  def find_password_reset_by_user_id_and_token
```

Comment:
I agree! I think I was just trying to follow the same naming convention as it was previously.